frontend/src/metabase/search/components/filters/CreatedByFilter/CreatedByContent.unit.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import userEvent from "@testing-library/user-event";
+import { useState } from "react";
+import { createMockUser } from "metabase-types/api/mocks";
+import type { User } from "metabase-types/api";
+import { screen, renderWithProviders, waitFor } from "__support__/ui";
+import type { CreatedByFilterProps } from "metabase/search/types";
+import { setupUsersEndpoints } from "__support__/server-mocks";
+import { CreatedByContent } from "./CreatedByContent";
+
+const TEST_USERS: User[] = [
+  createMockUser({ id: 1, common_name: "Alice" }),
+  createMockUser({ id: 2, common_name: "Bob" }),
+];
+
+const TestCreatedByContent = ({ onChange }: { onChange: jest.Func }) => {
+  const [value, setValue] = useState<CreatedByFilterProps>([]);
+  const onUserChange = (value: CreatedByFilterProps) => {
+    setValue(value);
+    onChange(value);
+  };
+  return <CreatedByContent value={value} onChange={onUserChange} />;
+};
+
+const setup = async ({ users = TEST_USERS } = {}) => {
```

Comment:
`setup` is never called with arguments, so we can simplify it and remove the argument